### PR TITLE
[CHERRY-PICK] Bugfix/mes 4765 fixing vehicle checks trailer calculation (#1238)

### PR DESCRIPTION
**Pull request by amobarak**
* Bug fix for trailer vehicle checks calculation fof serious faults for cat C and D

* Fixed test and removed unused tellMeQuestions in WRTC page

* Added tests for fault count helpers

## Checklist

- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Code has been tested manually
- [ ] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: src/providers/fault-count/cat-c/__tests__/fault-count.spec.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    });
+    it('2 driving faults result in 2 driving faults and no serious', () => {
+      expect((FaultCountCHelper as any)
+        .getVehicleChecksFaultCountCatC1(vehicleChecksTwoFaults).drivingFaults).toEqual(2);
+      expect((FaultCountCHelper as any)
+        .getVehicleChecksFaultCountCatC1(vehicleChecksTwoFaults).seriousFaults).toEqual(0);
+    });
+  });
+
+  describe('getVehicleChecksFaultCountCatE', () => {
```
Comment (TomBillingtonUK) [inline src/providers/fault-count/cat-c/__tests__/fault-count.spec.ts]: Typo we don't have a Cat E

File: src/providers/fault-count/cat-c/__tests__/fault-count.spec.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    it('2 driving faults result in 1 driving faults and 1 serious', () => {
+      expect((FaultCountCHelper as any)
+        .getVehicleChecksFaultCountCatCE(vehicleChecksTwoFaults).drivingFaults).toEqual(1);
+      expect((FaultCountCHelper as any)
+        .getVehicleChecksFaultCountCatCE(vehicleChecksTwoFaults).seriousFaults).toEqual(1);
+    });
+  });
+
+  describe('getVehicleChecksFaultCountCat1E', () => {
```
Comment (TomBillingtonUK) [inline src/providers/fault-count/cat-c/__tests__/fault-count.spec.ts]: Type we don't have a cat 1E

File: src/providers/fault-count/cat-d/__tests__/fault-count.spec.ts [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    });
+    it('2 driving faults result in 2 driving faults and no serious', () => {
+      expect((FaultCountDHelper as any)
+        .getVehicleChecksFaultCountCatD1(vehicleChecksTwoFaults).drivingFaults).toEqual(2);
+      expect((FaultCountDHelper as any)
+        .getVehicleChecksFaultCountCatD1(vehicleChecksTwoFaults).seriousFaults).toEqual(0);
+    });
+  });
+
+  describe('getVehicleChecksFaultCountCatE', () => {
```
Comment (TomBillingtonUK) [inline src/providers/fault-count/cat-d/__tests__/fault-count.spec.ts]: What's Count E

File: src/providers/fault-count/cat-d/__tests__/fault-count.spec.ts [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    it('2 driving faults result in 1 driving faults and 1 serious', () => {
+      expect((FaultCountDHelper as any)
+        .getVehicleChecksFaultCountCatDE(vehicleChecksTwoFaults).drivingFaults).toEqual(1);
+      expect((FaultCountDHelper as any)
+        .getVehicleChecksFaultCountCatDE(vehicleChecksTwoFaults).seriousFaults).toEqual(1);
+    });
+  });
+
+  describe('getVehicleChecksFaultCountCat1E', () => {
```
Comment (TomBillingtonUK) [inline src/providers/fault-count/cat-d/__tests__/fault-count.spec.ts]: Cat 1E?

**Review by TomBillingtonUK — APPROVED** [review]

**Review by appleabs — APPROVED** [review]

**Pull request merged**